docs/configuration/metrics/network-interfaces.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: default
+title: Azure Network Interface Declaration
+---
+
+## Azure Network Interface
```

Comment:
Thought - Should we elaborate on network interface vs network? For me this is not clear as a non-infra person but I think they are the same or not?

Reply:
I added a link to Microsoft docs for network interface.